File: existant/gestFormationWeb/gestion_formation_web/dto/DtoFormationCursus.cs
using gestion_formation_web.dao;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace gestion_formation_web.dto
{
    public class DtoFormationCursus
    {
        public static IEnumerable<formation_cursus> Get(int idCursus)
        {
            return DaoFormationCursus.Get(idCursus);
        }
        public static formation_cursus Get(int idCursus, int idFormation)
        {
            return DaoFormationCursus.Get(idCursus, idFormation);
        }
        public static formation_cursus GetByOrder(int idCursus, int ordre)
        {
            return DaoFormationCursus.GetByOrder(idCursus, ordre);
        }
        public static void Modifier(formation_cursus formationCursus)
        {
            DaoFormationCursus.Modifier(formationCursus);
        }
    }
}

File: Gestionnaire de formation/src/main/java/fr/esic/solutec/dto/DtoFormationCursus.java
package fr.esic.solutec.dto;

import java.util.List;

import javax.persistence.Query;

import fr.esic.solutec.dao.DaoFormationCursus;
import fr.esic.solutec.entities.FormationCursus;

public class DtoFormationCursus {

	
	
	public static FormationCursus getFormationCursus(int idFormationCursus) {
		
		return DaoFormationCursus.getFormationCursus(idFormationCursus);
	}
	
	public static List<FormationCursus> getListFormationCursus() {
		
		return DaoFormationCursus.getListFormationCursus();
	}
	
	
	public static void AddFormationCursus(FormationCursus formationCursus) {

		DaoFormationCursus.AddFormationCursus(formationCursus);
		
	}
	
	public static void EditFormationCursus(FormationCursus formationCursus) {
		DaoFormationCursus.EditFormationCursus(formationCursus);
	}
	
	
	public static List<FormationCursus> getListFormationCursusByIdCursus(int idCursus)
	{		
		return DaoFormationCursus.getListFormationCursusByIdCursus(idCursus);
	}
	
	
	public static List<Integer> getListOrdreByIdCursus(int idCursus) {
		return DaoFormationCursus.getListFormationOrdreByIdCursus(idCursus);
	}
	
	public static FormationCursus GetFormationCursusByOrdre(int idCursus, int ordre) {
		return DaoFormationCursus.GetFormationCursusByOrdre(idCursus, ordre);
		
	}
	
	public static void AjoutFormationFromCursus(FormationCursus formationCursus) {
		DaoFormationCursus.AjoutFormationFromCursus(formationCursus);
	}
	
}
